concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -110,6 +110,7 @@ datadog_api_key: ${datadog_api_key:-}
 datadog_app_key: ${datadog_app_key:-}
 enable_datadog: ${ENABLE_DATADOG}
 enable_cve_notifier: ${ENABLE_CVE_NOTIFIER:-false}
+enable_cve_monitor: ${ENABLE_CVE_MONITOR:-false}
```

Comment:
From what I see, we do not pass the `-var-file` for each environment when calling terraform: https://github.com/alphagov/paas-cf/blob/ace39811b938419861b3f43b0d04df808680bb4a/concourse/pipelines/create-bosh-cloudfoundry.yml#L2033-L2035